src/onegov/town6/forms/settings.py

```diff
@@ -143,5 +172,71 @@ def populate_chat_staff(self):
             (v, k) for v, k in staff_members
         ]
 
+    def validate(self):
+        result = super().validate()
+        if self.specific_opening_hours.data:
```

Comment:
Small nitpick, but you could save the result of `json_to_time` so we don't have to compute it twice.